app/screens/search/search.js

```diff
@@ -87,7 +91,9 @@ class Search extends PureComponent {
     componentDidMount() {
         this.setManagedConfig();
         if (this.refs.searchBar) {
-            this.refs.searchBar.focus();
+            setTimeout(() => {
+                this.refs.searchBar.focus();
+            }, 500);
```

Comment:
Is it possible to reduce this a bit more to around maybe 150ms? That should keep this feeling snappy

Reply:
I'll play with this value, but this is the best I came up with and it works every time, and if you tested it, you can see is really smooth